src/app/pages/update-profile-request/components/update-action.tsx

```diff
@@ -0,0 +1,83 @@
+import { useCallback, useState } from 'react';
+import { Button, Stack } from '@stacks/ui';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { delay } from '@app/common/utils';
+import { useCurrentAccount } from '@app/store/accounts/account.hooks';
+import { useUpdateProfileRequestSearchParams } from '@app/store/profiles/requests.hooks';
+import { UpdateProfilePayload } from '@app/common/profiles/requests';
+import { finalizeUpdateProfile } from '@shared/actions/finalize-profile-update';
+import { PublicProfile } from '@shared/profiles/types';
+import { createWalletGaiaConfig } from '@stacks/wallet-sdk';
+import { getHubInfo } from '../wallet-sdk-clone/utils';
+import { DEFAULT_PROFILE, fetchAccountProfileUrl, fetchProfileFromUrl, signAndUploadProfile } from '../wallet-sdk-clone/profiles';
```

Comment:
DEFAULT_PROFILE, fetchAccountProfileUrl, fetchProfileFromUrl, signAndUploadProfile and getHubInfo should be exported by wallet-sdk, or a combine method usable in useUpdateProfileSoftwareWallet

Reply:
wallet-sdk has been updated